styles.css

```diff
@@ -0,0 +1,95 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    margin: 0;
+    background-color: #7864ed;
+}
+
+header {
+    position: sticky;
+    display:flex;
+    justify-content: space-between;
+    height: 150px;
+    top: 0;
+    background-color: #6495ED;
+}
+
+h1 {
+    position: relative;
+    bottom: 75px;
+    font-family: 'Scheherazade New', serif;
+    font-size: 126px;
+    color: #0d02cf;
+    text-shadow: #000 0 0 1px,   #000 0 0 1px,   #000 0 0 1px,
+    #000 0 0 1px,   #000 0 0 1px,   #000 0 0 1px;
+}
+
+.hidden {
+    visibility: hidden;
+}
+
+button {
+    height: 150px;
+    width: 150px;
+    font-size: 64px;
+}
+
+section {
+    margin-top: 100px;
+    display: flex;
+    flex-flow: wrap row;
+    justify-content: space-around;
+}
+
+div {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    border: 3px solid #fff;
+    margin: 0 5px 50px 5px;
+    font-family: 'Inknut Antiqua', serif;
+    font-size: 18px;
+}
+
+img {
+    max-height: 500px;
+    max-width: 500px;
+}
+
+p {
+    color: #fff;
+    margin: 0;
+}
+
+@media (max-width: 1385px) {
+    h1 {
+        font-size: 64px;
+        position: static;
+    }
+}
+
+@media (max-width: 860px) {
+    .hidden {
+        display: none;
+    }
+}
+
+@media (max-width: 705px) {
+    h1 {
+        position: relative;
+        bottom: 40px;
+        left: 15px;
+    }
+}
+
+@media (max-width: 520px) {
```

Comment:
I would add to this media query a change in height/font size to your header as it goes onto 3 lines @ 464px